<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsWindow</class>
 <widget class="QWidget" name="ResultsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ResultsWindow</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="headerLabel">
       <property name="text">
        <string>Ergebnisse</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="resultsTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="OpenGLWindow" name="openGLWindow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="progressSlider">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OpenGLWindow</class>
   <extends>QOpenGLWidget</extends>
   <header>qt/openglwindow.h</header>
   <slots>
    <slot>OnPlayButtonPressed()</slot>
    <slot>OnPauseButtonPressed()</slot>
    <slot>OnStopButtonPressed()</slot>
    <slot>OnProgressSliderValueChanged(int)</slot>
    <slot>OnTPoseButtonPressed()</slot>
    <slot>OnToggleSkeleton()</slot>
    <slot>OnToggleTrackerOffsets()</slot>
    <slot>OnTogglePlay()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>progressSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>openGLWindow</receiver>
   <slot>OnProgressSliderValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1043</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>1043</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>pressed()</signal>
   <receiver>openGLWindow</receiver>
   <slot>OnStopButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1042</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>897</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resultsTable</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>ResultsWindow</receiver>
   <slot>CellSelected(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>openGLWindow</receiver>
   <slot>OnTogglePlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CellSelected(int,int)</slot>
  <slot>OnSaveResultsButtonPressed()</slot>
 </slots>
</ui>
